platform/src/main/java/org/hillview/table/rows/VirtualRowHashStrategy.java

```diff
@@ -0,0 +1,52 @@
+package org.hillview.table.rows;
+
+import it.unimi.dsi.fastutil.ints.Int2ObjectMap;
+import it.unimi.dsi.fastutil.ints.Int2ObjectOpenCustomHashMap;
+import it.unimi.dsi.fastutil.ints.IntHash;
+import it.unimi.dsi.fastutil.objects.Object2IntOpenHashMap;
+import it.unimi.dsi.fastutil.objects.ObjectIterator;
+import org.hillview.table.Schema;
+import org.hillview.table.api.ITable;
+import org.hillview.utils.MutableInteger;
+
+public class VirtualRowHashStrategy implements IntHash.Strategy {
```

Comment:
This kind of class really needs a comment, otherwise no one would know how to use it.